metricbeat/docs/developer-guide/create-metricset.asciidoc

```diff
@@ -1,45 +1,45 @@
+A metricset is the part of a Metricbeat module that fetches and structures the
+data from the remote service. Each module can have multiple metricsets.
 
-This guide should help you to create your own MetricSet. It will focus on the details
-of the MetricSet creation itself. If you want to create your own beat using Metricbeat
-as a library, have a look here.
+In this guide, you learn how to create your own metricset. If you want to create
+your own Beat that uses Metricbeat as a library, see <<creating-beat-from-metricbeat>>.
 
-In general when creating a MetricSet for the first time, it helps before to have a look at the
-implementation of the existing MetricSets as inspiration.
+When creating a metricset for the first time, it generally helps to look at the
+implementation of existing metricsets for inspiration.
 
 
 [float]
-== Create Metricset Files
-Creating a new metricset is simple. Run the command
+=== Creating Metricset Files
+Creating a new metricset is simple. Run the following command inside your beat directory:
 
 [source,bash]
 ----
 make create-metricset
 ----
 
-inside your beat directory. This will prompt you to enter a module and metricset name. Only use characters `[a-z]` here
-and if required also `_`. No other characters are allowed.
+You'll be prompted to enter a module and metricset name. Only use characters `[a-z]` 
+and, if required, underscores (`_`). No other characters are allowed.
 
-This will create all basic files for your metricset and if the module did not exist yet, it will also add the required
-module files. More details on the module files can be found here (TODO: link to module guide).
+When you run `make create-metricset`, it creates all the basic files for your metricset, along with the required module
+files if the module does not already exist. See <<creating-metricbeat-module>> for more details about the module files.
 
-NOTE: We will used `{metricset}`, `{module}`, `{beat}` across this guide. It means this part should be replaced by the actual
-name of your metricset, module or beat.
+NOTE: We use `{metricset}`, `{module}`, and `{beat}` in this guide as placeholders. You need to replace these with
+the actual names of your metricset, module, and beat. 
 
-The create metricset is already a functioning metricset and can be compiled. To compile your new metricset, run the commands:
+The metricset that you created is already a functioning metricset and can be compiled. To compile your new metricset, run the following:
 
 [source,bash]
 ----
 make collect
 make
 ----
 
-The first command `make collect` will update all generated files with the most recent metricset, the second command `make`
-will compile your source code and provide your with a binary in the beat folder calle `{beat}`. The binary can be run
-in debug mode with
+The first command, `make collect`, updates all generated files with the most recent metricset. The second command,
```

Comment:
with the most recent files from the metriceset
with the most recent data from the metriceset
with the most recent meta information from the metriceset